lib/constants.js

```diff
@@ -1,25 +1,35 @@
 'use strict';
 Object.defineProperty(exports, "__esModule", { value: true });
-class KubernetesWorkload {
-}
-KubernetesWorkload.pod = 'Pod';
-KubernetesWorkload.replicaset = 'Replicaset';
-KubernetesWorkload.deployment = 'Deployment';
-KubernetesWorkload.statefulSet = 'StatefulSet';
-KubernetesWorkload.daemonSet = 'DaemonSet';
-KubernetesWorkload.job = 'job';
-KubernetesWorkload.cronjob = 'cronjob';
+exports.workloadTypesWithRolloutStatus = exports.workloadTypes = exports.deploymentTypes = exports.ServiceTypes = exports.DiscoveryAndLoadBalancerResource = exports.KubernetesWorkload = void 0;

```

Comment:
Did you make any changes to `constants.ts` file? Or this convention was changed due to a compiler version diff?